spec/setup_test_database.rb

```diff
@@ -3,7 +3,7 @@
 def setup_test_database
   p "Setting up test database..."
 
-  DatabaseConnection.setup('makers_bnb_test')
-  DatabaseConnection.query('TRUNCATE users, spaces, bookings CASCADE;')
-  # DatabaseConnection.query("ALTER SEQUENCE users_id_seq RESTART WITH 1;")
+  connection = PG.connect(dbname: 'makers_bnb_test')
+  connection.exec('TRUNCATE users CASCADE;')
+  # Not how it's meant to be done - when merging, please keep original DatabaseConnection!
```

Comment:
Maybe one to discuss. What do you mean here pls?

Reply:
We just didn't make the Database Connection file which you guys have, but we can edit this once merged!

Reply:
this was just a note to say we need to create our DatabaseConnection class to neaten this up as you did in yours

Reply:
Ahh great, thanks for clarifying!